src/main/java/pl/equipRental/assets/AssetController.java

```diff
@@ -0,0 +1,25 @@
+package pl.equipRental.assets;
+
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping("/api/assets")
+public class AssetController {
+
+    private final AssetService assetService;
+
+    @GetMapping("")
+    public List<AssetDto> findAll() {
+        return assetService.findAll();
```

Comment:
Czy nie warto jest użyć Pageable do pejdżowania ?

Reply:
to zostawiamy na potem